### ISSUE TITLE
Highlighting in drop downs is either missing or 'white on white' and unreadable

### ISSUE THREAD

**Issue by jim618** [bitcoin-solutions/multibit-hd#606]
In the default colouring scheme the drop down wizards have a couple of problems:

+ On the Credentials screen there is no highlight when you mouse over wallets:
   Screenshot:
![screen shot 2015-06-15 at 14 09 58](https://cloud.githubusercontent.com/assets/919427/8160582/bd41e0f4-1368-11e5-8677-2740e1b2a58b.png)

(mouse was hovering in drop down list - not shown on the screenshot)
+ And on a send when you select a Recipient the highlighting is 'white on white'.

   Screenshot:

![screen shot 2015-06-15 at 14 11 36](https://cloud.githubusercontent.com/assets/919427/8160586/cc141dfe-1368-11e5-8bfc-bf109c738fd8.png)

These appear to have been introduced when we bumped the version of Java just prior to Release 0.1 but that could be a coincidence

**Comment by gary-rowe** [comment]
This was caused by inconsistent implementation of `ListCellRenderer` and `TableCellRenderer`. Each now has a specific reference to the underlying theme `Color` setting for all combinations:

![screen shot 2015-06-15 at 22 21 32](https://cloud.githubusercontent.com/assets/342133/8171227/4ce54a94-13ae-11e5-82bb-a627bba5dccd.png)

![screen shot 2015-06-15 at 22 25 13](https://cloud.githubusercontent.com/assets/342133/8171205/2b50087e-13ae-11e5-87af-7ee8c25fa722.png)

In Dark theme everything went a bit strange and various inconsistencies really showed up. In the end I had to change the sidebar/table selection `Color` setting to a contrasting value from the data entry yellow. The result looks like this:

![screen shot 2015-06-15 at 22 50 48](https://cloud.githubusercontent.com/assets/342133/8172258/ad3bd360-13b6-11e5-948b-cb8063f6bfb2.png)

I'm OK with the change from yellow to blue in the Dark theme since it works better in the low light conditions that it's designed for (less glare from the large yellow selection bar).

To verify it is best to try out various drop down lists and tables throughout in different themes. They should all be consistent and there should be no spurious blocks of a different colour (such as image or check box backgrounds in highlighted rows).

Ready for review and close.
